### PR TITLE
Task/01 setup

### PR DESCRIPTION
**Pull request by VasylBiliak**

### CODE REVIEW

**Comment by VasylBiliak** [comment]
- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [ ]  On the widths 390px and 1280px, the site layout corresponds to the design

**Review by MZabolotnev — CHANGES_REQUESTED** [review]
Pls check deployment - its failed

File: src/components/about/About.component.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import styles from './about.module.css';
+
+export function AboutComponent() {
+    return (
+        <div className={styles.about}>
+            <div className={styles.content}>
+                <img src={picture} alt="About" className={styles.picture} />
+                <div className={styles.content}>
+                    <h1 className={styles.title}>About me</h1>
+                    <div className={styles.description}>
```
Comment (MZabolotnev) [inline src/components/about/About.component.tsx]: It can be more semantic. Pls use section / article instead of div

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

**Review by MZabolotnev — CHANGES_REQUESTED** [review]
Paddings missed in both views ( mobile / desktop )
image

**Review by MZabolotnev — CHANGES_REQUESTED** [review]
Still not good. Reason - picture width is hardcoded width: 330px;. You can try width: 100% instead. 
image

**Review by MZabolotnev — APPROVED** [review]

**Pull request merged**